Pedantic compiler warnings

PR #228 introduced a large number of compiler warnings caused by an extra semi-colon:

> In file included from /vagrant/build/src/ui_recipeExtrasWidget.h:13:0,
                 from /vagrant/src/RecipeExtrasWidget.h:31,
                 from /vagrant/src/RecipeExtrasWidget.cpp:25:
/vagrant/src/BtLineEdit.h:51:105: warning: extra ‘;’ [-Wpedantic]
    Q_PROPERTY( int     type              READ type              WRITE setType              STORED false);
                                                                                                         ^
/vagrant/src/BtLineEdit.h:52:105: warning: extra ‘;’ [-Wpedantic]
    Q_PROPERTY( QString configSection     READ configSection     WRITE setConfigSection     STORED false);
                                                                                                         ^
/vagrant/src/BtLineEdit.h:53:105: warning: extra ‘;’ [-Wpedantic]
    Q_PROPERTY( QString editField         READ editField         WRITE setEditField         STORED false);
                                                                                                         ^
/vagrant/src/BtLineEdit.h:54:105: warning: extra ‘;’ [-Wpedantic]
    Q_PROPERTY( QString forcedUnit        READ forcedUnit        WRITE setForcedUnit        STORED false);
                                                                                                         ^
/vagrant/src/BtLineEdit.h:55:105: warning: extra ‘;’ [-Wpedantic]
    Q_PROPERTY( QString forcedScale       READ forcedScale       WRITE setForcedScale       STORED false);

--- a/src/BtLineEdit.h
+++ b/src/BtLineEdit.h
@@ -48,11 +48,11 @@ class BtStringEdit;
 class BtLineEdit : public QLineEdit
 {
    Q_OBJECT
-   Q_PROPERTY( int     type              READ type              WRITE setType              STORED false);
-   Q_PROPERTY( QString configSection     READ configSection     WRITE setConfigSection     STORED false);
-   Q_PROPERTY( QString editField         READ editField         WRITE setEditField         STORED false);
-   Q_PROPERTY( QString forcedUnit        READ forcedUnit        WRITE setForcedUnit        STORED false);
-   Q_PROPERTY( QString forcedScale       READ forcedScale       WRITE setForcedScale       STORED false);
+   Q_PROPERTY( int     type              READ type              WRITE setType              STORED false)
+   Q_PROPERTY( QString configSection     READ configSection     WRITE setConfigSection     STORED false)
+   Q_PROPERTY( QString editField         READ editField         WRITE setEditField         STORED false)
+   Q_PROPERTY( QString forcedUnit        READ forcedUnit        WRITE setForcedUnit        STORED false)
+   Q_PROPERTY( QString forcedScale       READ forcedScale       WRITE setForcedScale       STORED false)
 
 public:
 

